app/controllers/admin/users_controller.rb

```diff
@@ -31,6 +31,7 @@ class UsersController < Admin::ApplicationController
 
     def index
       @users = Admin::UsersQuery.call(
+        relation: User.includes([:organizations]),
```

Comment:
I noticed with this change we're overriding the default `UsersQuery` relation which is `User.registered`, and I can see invited users popping up in the main members index now - maybe just a small tweak needed there?

```suggestion
        relation: User.registered.includes([:organizations]),
```

Reply:
Well, that's unfortunate! I checked for this bullet warning on desktop and on mobile and it didn't show for me. 🙃 

Let me update this.